Project.toml

```
@@ -62,5 +62,5 @@ StaticArrays = "1"
 StrideArrays = "0.1"
 TimerOutputs = "0.5.25"
 TrixiBase = "0.1.5"
-WriteVTK = "1"
+WriteVTK = "^1.21.2"
```

```suggestion
WriteVTK = "1.21.2"
```
Redundant `^` (this is the default).